src/box/lua/info.c

```diff
@@ -650,6 +663,22 @@ lbox_schema_version(struct lua_State *L)
 	return 1;
 }
 
+/* gethostname() Lua interface inside box.info. */
```

Comment:
Nit: function/struct/field is usually described using a documentation-style comment: `/** <...> */`.

Reply:
I would prefer to remove this comment entirely, as the nearby functions don't have one and the code is self-explanatory, but checkpatch is opposed to that. Otherwise, added extra `*`.

Reply:
> added extra `*`.

Forgot to push?